### ISSUE TITLE
Cant add dropout in ConvLSTM2D

### ISSUE THREAD

**Issue by yaxiongma** [keras-team/keras#11102]
Hi there,
     ###  In order to alleviate the over fitting problem existing in my model, I consider to add some drop out.
     ###  The model being used is **ConvLSTM2D** .  But unfortunately to find I can't do this via  adding an item 
**recurrent_dropout = 0.35**. 
### Here are my model configure and error log:
`
seq = Sequential()

seq.add(ConvLSTM2D(filters=1, strides=1,kernel_size=(3, 1),
                   input_shape=(10,128, 128, 1),
                   kernel_regularizer=regularizers.l2(0.01),
                   activation = 'relu',
                   return_sequences=True),
                   recurrent_dropout = 0.35
                   #dropout = 0.3#this will also result in the same error as adding a recurrent_dropout
                   )
`
**More Info**: I use keras 2.2.2 with tensorflow 1.9.0. And, there is reference about keras documentation
> https://keras.io/layers/recurrent/#convlstm2d  
`
keras.layers.ConvLSTM2D(filters, kernel_size, strides=(1, 1), padding='valid', data_format=None, dilation_rate=(1, 1), activation='tanh', recurrent_activation='hard_sigmoid', use_bias=True, kernel_initializer='glorot_uniform', recurrent_initializer='orthogonal', bias_initializer='zeros', unit_forget_bias=True, kernel_regularizer=None, recurrent_regularizer=None, bias_regularizer=None, activity_regularizer=None, kernel_constraint=None, recurrent_constraint=None, bias_constraint=None, return_sequences=False, go_backwards=False, stateful=False, dropout=0.0, recurrent_dropout=0.0)
`

thanks a lot!

**Comment by yanboliang** [comment]
@yaxiongma  I think you should move forward ```recurrent_dropout = 0.35``` to ahead of the ``)```, it's one of the arguments in ```ConvLSTM2D```, not ```Sequential.add()```.

**Comment by yaxiongma** [comment]
@yanboliang  
thanks a lot,Yanbo . It works. I should have checked it carefully
